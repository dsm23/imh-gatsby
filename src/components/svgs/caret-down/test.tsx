/// <reference types="jest" />

import React from 'react';
import { render } from '@testing-library/react';

import CaretDown from '.';

const renderable = <CaretDown />;

describe(`CaretDown`, () => {
  it(`it renders`, () => {
    const { baseElement } = render(renderable);

    expect(baseElement).toBeTruthy();
  });

  it(`matches snapshot`, () => {
    const { asFragment } = render(renderable);

    expect(asFragment()).toMatchSnapshot();
  });
});
