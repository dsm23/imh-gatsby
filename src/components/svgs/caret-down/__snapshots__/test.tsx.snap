// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaretDown matches snapshot 1`] = `
<DocumentFragment>
  <svg
    fill="#f4f5f7"
    viewBox="0 0 20 20"
  >
    <path
      clip-rule="evenodd"
      d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
      fill-rule="evenodd"
    />
  </svg>
</DocumentFragment>
`;
